systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlST.java

```diff
@@ -302,23 +298,17 @@ void testCruiseControlIntraBrokerBalancingWithoutSpecifyingJBODStorage(Extension
         final TestStorage testStorage = new TestStorage(extensionContext);
 
         resourceManager.createResource(extensionContext, KafkaTemplates.kafkaWithCruiseControl(testStorage.getClusterName(), 3, 3).build());
-        resourceManager.createResource(extensionContext, KafkaRebalanceTemplates.kafkaRebalance(testStorage.getClusterName())
+        resourceManager.createResource(extensionContext, false, KafkaRebalanceTemplates.kafkaRebalance(testStorage.getClusterName())
             .editOrNewSpec()
                 .withRebalanceDisk(true)
             .endSpec()
             .build());
 
-        final String coPodName = KubeClusterResource.kubeClient(INFRA_NAMESPACE).getClusterOperatorPodName();
-        final String coLog = StUtils.getLogFromPodByTime(INFRA_NAMESPACE, coPodName, STRIMZI_DEPLOYMENT_NAME, "60s");
-
-        // CO logs must contain warning message
-        assertThat(coLog, CoreMatchers.containsString("Intra-broker balancing only applies to Kafka deployments that use JBOD storage with multiple disks."));
+        final String errMessage = "Intra-broker balancing only applies to Kafka deployments that use JBOD storage with multiple disks.";
```

Comment:
This string doesn't seem to be used for anything? Is is used by the test framework?

Reply:
I think it can be removed :) If such string is not present in the status of the CR.

Reply:
Removed